proposals/lip-0028.md

```diff
@@ -31,37 +31,37 @@ When a transaction object is serialized, also the asset property has to be seria
 
-As it is shown in the diagram below, every transaction object in the Lisk protocol, regardless of the type, has a common set of 6 properties. The data related to the specific transaction type is given in the `asset` property. This architecture is also expected to be used for custom transactions created with the Lisk SDK. For simplicity and consistency, custom transactions should have these 6 properties, while all the new logic and data is defined with respect to the custom `asset` property.
+As it is shown in the diagram below, every transaction object in the Lisk protocol, regardless of the type, has a common set of 7 properties. The data related to the specific transaction type is given in the `asset` property. This architecture is also expected to be used for custom transactions created with the Lisk SDK. For simplicity and consistency, custom transactions should have these 7 properties, while all the new logic and data is defined with respect to the custom `asset` property.
 
 ![Transaction Serialization Diagram](lip-0028/transaction_serialization_diagram.png)
 
-For this reason, this LIP proposes to divide the serialization-deserialization process. First it defines a serialization-deserialization process for the whole transaction object with the `baseTransaction` JSON schema where the `asset` property is assumed to be of type `bytes`. Secondly, it defines the serialization-deserialization process for the `asset` property with the corresponding `asset` property JSON schema depending on the type of the transaction (specified in the `type` property). There are 7 different `asset` property schemas, one for each of the transaction types in the Lisk protocol. In order to simplify the creation of custom transactions with the SDK and to make the validation process less error prone, every property defined inside a given `asset` schema is required.
+For this reason, this LIP proposes to divide the serialization-deserialization process. First it defines a serialization-deserialization process for the whole transaction object with the `transactionSchema` JSON schema where the `asset` property is assumed to be of type `bytes`. Secondly, it defines the serialization-deserialization process for the `asset` property with the corresponding `asset` property JSON schema depending on the type of the transaction (specified by the unique combination of `moduleID` and `assetID` properties). There are 7 different `asset` property schemas, one for each of the transaction types in the Lisk protocol. In order to simplify the creation of custom transactions with the SDK and to make the validation process less error prone, every property defined inside a given `asset` schema is required.
```

Comment:
> specified by the unique combination of `moduleID` and `assetID` properties

I would say "given by the values of  `moduleID` and `assetID`".

Reply:
changed